pages/samvera/developer_resources/collections/hyrax_2.1/participants.md

```diff
@@ -0,0 +1,203 @@
+* When the create new collection process is initiated, the user is allowed to create collections of types for which they have create access.
+* When a collection type creator creates a new collection, the creator is made a manager of only that collection which grants them `edit_access` to the new collection.  This `edit_access` grant is what grants the creator the abilities to edit the collection, collection items, and metadata.
+
+NOTE: Access to a collection is granted when the collection is created.  Changes to creators after a collection is created do not change the access grants of the collection.  These changes only effect who can create new collections in the future.
+
+NOTE: A site can set creators of Admin Set collection type, which gives these creators the ability to create new admin sets.
+
+### Configuring collection sharing
+
+The collection type controls whether collections of that type can be shared.  If Sharable? is true, then collections of this type can be shared; otherwise, they can not.  Collections that can be shared have a Sharing tab in the collection edit form.
+
+*Proposed, but not yet implemented:* For backward compatibility, the sharable setting should have additional configuration that determines how participants are applied to works created directly in the collection.
+
+- [x] Collection managers are given `edit access` to the collection (cannot be turned off)
+- [ ] Collection viewers are given `read access` to the collection which allows them to see works in the collection regardless of the works visibility.
+- [ ] Works are assigned participant access where managers are given edit_access and viewers are given read_access to any work created directly in the collection.
+
+
+These additional settings allow for backward compatibility for pre-defined collection types.
+
+* User Collection type shares permissions for the collection only, but does not apply permissions to works.
+* Admin Set type does not share view permissions to the admin set, but does apply permissions to works.
+* The default is for all to be checked.
+
+
+----
+
+## Understanding Collection Participants
+
+### How to set participants for a collection?
+
+* Dashboard -> Collections
+* Click Edit collection in action menu beside the collection you want to update
+* Select Sharing tab
+* Search for user/group
+* Set access level to Manager or Depositor or Viewer
+* Click Add button
+
+
+### Manager
+
+Managers of a collection can add to and remove works from the collection, modify collection metadata, and delete the collection.  If configured to set work permissions, the managers are given 'edit_access' to any work created directly in the collection.
+
+**Default:**  user creating the collection, Collection Type managers for this collection's type
+
+**Implementation:**
+
+* When a manager is added
+  * an entry is added to `permission_template_accesses` database table granting the user/group :manage access 
```

Comment:
Here you talk about the table name, but below you talk about `Hydra::AccessControls::Permission`, a class name.  How about using `Hyrax::PermissionTemplateAccess` here for consistency?

Reply:
This was updated in collection-sharing.md